components/Header/NavHeader/NavHeader.vue

```diff
@@ -120,12 +120,6 @@ export default {
   cursor: pointer;
 }
 
```

Comment:
This wasn't used anywhere so removed

Reply:
did we delete them everywhere or did we just delete them from the .css master file? because in some old components it is still locally defined. It'd be worth a find-all sweep of the code base to delete any of them